src/fluidplayer.js

```diff
@@ -1910,41 +1911,48 @@ const fluidPlayerClass = function () {
         };
         let initiateVolumebarTimerId = setInterval(initiateVolumebar, 100);
         self.destructors.push(() => clearInterval(initiateVolumebarTimerId));
-
-        if (self.displayOptions.layoutControls.doubleclickFullscreen && !(self.isTouchDevice() || self.displayOptions.layoutControls.controlForwardBackward.doubleTapMobile)) {
```

Comment:
The issue lies in this `if()` here. There's a problem with one of the conditions, so I would check in here

Reply:
Ok, ill revert the prettier part, remove the logs, and fix the initialization happening 2x 👍